src/gffread/config.vsh.yaml

```diff
@@ -0,0 +1,396 @@
+name: gffread
+namespace: gffread
+description: Validate, filter, convert and perform various other operations on GFF files.
+keywords: [gff, conversion, validation, filtering]
+links:
+  homepage: https://ccb.jhu.edu/software/stringtie/gff.shtml#gffread
+  documentation: https://ccb.jhu.edu/software/stringtie/gff.shtml#gffread
+  repository: https://github.com/gpertea/gffread
+references: 
+  doi: 10.12688/f1000research.23297.2
+license: MIT
+requirements:
+  commands: [ gffread ]
+argument_groups:
+  - name: Inputs
+    arguments:
+      - name: --input
+        type: file 
+        direction: input
+        description: |
+          A reference file in either the GFF3, GFF2 or GTF format.
+        required: true
+        example: annotation.gff
+      - name: --chr_mapping
+        alternatives: -m
+        type: file
+        direction: input
+        description: |
+          <chr_replace> is a name mapping table for converting reference sequence names, 
+          having this 2-column format: <original_ref_ID> <new_ref_ID>.
+      - name: --seq_info
+        alternatives: -s
+        type: file
+        direction: input
+        description: |
+          <seq_info.fsize> is a tab-delimited file providing this info for each of the mapped 
+          sequences: <seq-name> <seq-length> <seq-description> (useful for -A option with 
+          mRNA/EST/protein mappings).
+      - name: --genome
+        alternatives: -g
+        type: file
+        description: |
+          Full path to a multi-fasta file with the genomic sequences for all input mappings, 
+          OR a directory with single-fasta files (one per genomic sequence, with file names 
+          matching sequence names).
+        example: genome.fa
+  - name: Outputs
+    arguments:
+      - name: --outfile
+        alternatives: -o
+        type: file
+        direction: output
+        must_exist: false
+        description: |
+          Write the output records into <outfile> instead of stdout.
```

Comment:
Make this a required argument -- the stdout produced by Viash components are only used for logging

Reply:
Can we give a default file name instead of making it required? So no output file needs to be specified and the usage  syntax stays more similar to the original gffread

Reply:
If the component needs to return this file in order to function properly, please make it a required output file. This way, we can let Nextflow (and users) know which output files are expected and which are optional. I'll make a suggestion :)